lib/rom/repository/command_compiler.rb

```diff
@@ -208,7 +208,10 @@ def finalize_command_class(klass, relation)
 
         klass.extend_for_relation(relation) if klass.restrictable
 
-        plugins.each { |plugin| klass.use(plugin) }
+        plugins.each do |(plugin, block)|
+          klass.use(plugin) 
+          klass.instance_eval(&block) if block
```

Comment:
@solnic oh, this is a dangerous road :) The problem is we're using a cache in `CommandCompiler` and two blocks will return the same `hash` value only if the are the same objects. In other words such code
```ruby
class MyRepo < ...
  def my_create(*args)
    command(..., use: { timestamps: proc { ... }})
  end
end
```
will cause a memory leak by bloating the cache